lib/utils/protocols/rtsp.js

```diff
@@ -35,6 +35,21 @@ const sessionId = (buffer) => {
   return val ? val.split(';')[0] : null
 }
 
+const sessionTimeout = (buffer) => {
+  const val = extractHeaderValue(buffer, 'Session')
+  if (val === null) {
+    return null
+  }
+  const timeoutToken = 'timeout='
+  const timeoutPosition = val.indexOf(timeoutToken)
+  if (timeoutPosition !== -1) {
+    let timeoutVal = val.substring(timeoutPosition + timeoutToken.length)
+    timeoutVal = timeoutVal.split(';')[0]
+    return parseInt(timeoutVal)
```

Comment:
It would be nice if you could return null if `parseInt` returns a `NaN`, so that the function always returns either `null` or an integer.